steward/src/cork.rs

```diff
@@ -87,7 +90,7 @@ impl steward::contract_call_server::ContractCall for CorkHandler {
                 "failed to send cork to sommelier",
             ));
         }
-        debug!("sent cork!");
+        info!("submitted cork for {}!", request.cellar_id);
```

Comment:
Could we decorate this log message further by including e.g. the type URL of the request, so we know what sort of message is being sent to the cellar? Same would apply to the message at the start of processing as well.

Reply:
That's a good idea. I was trying to figure out how to log what function was being called and I forgot about the type url.

Reply:
It looks like there is no proto type_url available unless it's a `prost::Any`. There is a Rust type_id which may accomplish the same thing.